ThingWall/Models/ItemViewModels.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web;
+
+namespace ThingWall.Models
+{
+    public class ItemViewModels
+    {
+        public int ItemID { get; set; }
+        public string Name { get; set; }
+        public string OwnerId { get; set; }
```

Comment:
Uwaga na pułapkę, string domyślnie może przyjmować nulla. Więc tworzy się wiązanie "item może posiadać 0 lub 1 właściciela", co oznacza, że przy usunięciu właściciela z bazy nie usuną się jego itemy. Skorzystaj z [Required] jeśli nie to było Twoją intencją.